ci-operator/config/openshift/installer/openshift-installer-fcos.yaml

```diff
@@ -58,7 +58,7 @@ images:
       paths: null
   to: baremetal-installer
 promotion:
-  name: "4.4"
+  name: "4.3"
```

Comment:
what are these for? should we not switch all OKD builds to 4.4 master now? we don't want to follow product branching there, but roll master essentially. cc @vrutkovs

Reply:
ah, not intentional.  Not sure how they got in here either.  Pulling them out now...

Reply:
Yeah, it needs to be reverted - OKD follows 4.4

Reply:
Pushed e8ad93b70 -> 562d6faf1 dropping the 4.4->4.3 stuff.